client/examples/README.md

```diff
@@ -138,3 +139,32 @@ This should give you following output:
+| init         | < client_path >                                                                                     | initializes the underlying Stronghold system                                                                                  |
+| keygen       | < key_type > < vault_path > < record_path >                                                         | Generates a key "ed25510" or "x25519", at location composed of "vault_path" and "record_path"                                 |
+| backup       | < path_to_snapshot_location > < passphrase >                                                        | Writes the current Stronghold state to "path_to_snapshot_location" (a path) with "passphrase"                                 |
+| restore      | < path_to_snapshot_location > < passphrase >                                                        | Loads the current Stronghold state from "path_to_snapshot_location" (a path) with "passphrase"                                |
+| slip10gen    | < vault_path > < record_path >                                                                      |                                                                                                                               |
+| slip10derive | < chain > < vault_path_origin > < record_path_origin > < vault_path_derive > < record_path_derive > | Derives a new key from old location and stores it into the new location. The key is being derive from the provided chain code |
+| mnemonic     | -                                                                                                   | Creates a BIP39 mnemonic and generates a seed                                                                                 |
+| bip39restore | -                                                                                                   |                                                                                                                               |
+| quit         | -                                                                                                   | Exits the REPL                                                                                                                |
+| checkrecord  | < vault_path > < record_path >                                                                      | Checks a record inside the vault, if it exists                                                                                |
+| checkvault   | < vault_path >                                                                                      | Checks for the presence of a vault                                                                                            |
+
+**Note:**
+Each command will show a short help message, if the entered command was incorrect.
```

Comment:
```suggestion
| Command      | Parameters                                                                                          | Description                                                                                                                    |
|:-------------|:----------------------------------------------------------------------------------------------------|:-------------------------------------------------------------------------------------------------------------------------------|
| help         | -                                                                                                   | Display a help message.                                                                                                        |
| init         | < client_path >                                                                                     | Initializes the underlying Stronghold system.                                                                                  |
| keygen       | < key_type > < vault_path > < record_path >                                                         | Generates a "ed25510" or "x25519" key at a location composed of "vault_path" and "record_path".                                |
| backup       | < path_to_snapshot_location > < passphrase >                                                        | Writes the current Stronghold state to "path_to_snapshot_location" (a path) with "passphrase".                                 |
| restore      | < path_to_snapshot_location > < passphrase >                                                        | Loads the current Stronghold state from "path_to_snapshot_location" (a path) with "passphrase".                                |
| slip10gen    | < vault_path > < record_path >                                                                      |                                                                                                                                |
| slip10derive | < chain > < vault_path_origin > < record_path_origin > < vault_path_derive > < record_path_derive > | Derives a new key from old location and stores it into the new location. The key will be derived from the provided chain code. |
| mnemonic     | -                                                                                                   | Creates a BIP39 mnemonic and generates a seed.                                                                                 |
| bip39restore | -                                                                                                   |                                                                                                                                |
| quit         | -                                                                                                   | Exits the REPL.                                                                                                                |
| checkrecord  | < vault_path > < record_path >                                                                      | Checks a record inside the vault.                                                                                              |
| checkvault   | < vault_path >                                                                                      | Checks for the presence of a vault.                                                                                            |

**Note:**
Each command will show a short help message, if the entered command or parameters are incorrect.
```

Reply:
btw, the description `checkrecord` seemed to hint that it would return the record if it exists or something similar. "Checks a record inside the vault, if it exists"  Should we add this?

Reply:
> "Checks a record inside the vault, if it exists"
yeah, i agree here. `checkrecord` returns "yes" or "no", but not the record itself. I will update this phrase.

Reply:
No, the record will not be returned, but "yes" or "no" will be displayed in case the record exists or not. Otherwise it would break the main principle of Stronghold.

Reply:
Updated the suggestion to reflect this.

Reply:
awesome :+1: